<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientIMPOnWidget</class>
 <widget class="QWidget" name="ClientIMPOnWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientIMPOnWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>连接状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cConnectState">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>未连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cBtnSend">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="cMsgHistory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>691</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cMsgForSend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
